src/status_im2/common/scroll_page/view.cljs

```diff
@@ -108,7 +104,7 @@
          sticky-header
          children]
       [:<>
-       [scroll-page-header @scroll-height name page-nav-right-section-buttons cover-image sticky-header]
+       [:f> scroll-page-header @scroll-height name page-nav-right-section-buttons cover-image sticky-header]
```

Comment:
could you elaborate pls ?

Reply:
`:f>` it is a way to create a functional React component. Only in functional component hooks can be used. And hooks were introduced for animations here - #14695.
So in my changes, it is as a result of rebase.

Reply:
yeah sorry wrong place, for comment, i wanted ask about `display-picture` why f was moved from function to outside usage of component, whats the point

Reply:
Okay, full story :)

Original reference to `display-picture` was:
```clojure
[display-picture scroll-height cover]
```
where `scroll-height` is an atom. And inside `display-picture` there were a bunch of dereferences of that atom.

I don't like passing atoms inside components. Because react already has idiomatic ways to get info in and out of components - props and callbacks. Also passing atom in the props makes changes of atom by child components uncontrolled. 

So I removed atoms dereferencing within `display-picture` and changed its call:
```clojure
[display-picture @scroll-height cover]
```

And flickering started.
At that point `display-picture` looked like this:
```clojure
(defn display-picture
  [scroll-height cover]
  [:f>
   (fn []
     (let [input-range (if platform/ios? [-67 10] [0 150])
...
```
It is a functional react component inside one more component. When atom used as an argument, it wasn't technically changed, so outer component wasn't re-rendering content. But now, when value of `scroll-height` used and changes constantly `display-picture` started to recreate internal functional react component all the time. Flickering as a result.
And since there is no any reason in additional wrapper component, I got rid of it, leaving a function itself.

And I believe `[:f> display-picture @scroll-height cover-image]` is quite convenient, it makes it obvious that we are dealing with functional component here.

Reply:
how a dev might know that ` display-picture` can be used only with `:f>` ? anyway mixing reactive atoms and hooks not a good idea, but its not a discussion for this PR

Reply:
`:f>` should definitely be inside component, and we need to find a proper solution for `scroll-height` probably it needs to be reimplemented

Reply:
Good question! I think docstring can be a good solution.

Reply:
Thanks for the detailed explanation @vkjr. I don't have much to add, but we can see in Reagent's unit tests and docs the usage of `:f>` outside the component. https://github.com/reagent-project/reagent/blob/master/doc/ReactFeatures.md#hooks

So to me this doesn't feel like a workaround in any way and I wouldn't even worry about the docstring. If we can, I agree we should use `:f>` inside components, but for maximum flexibility, as @vkjr demonstrated, using it outside can have benefits too.

Reply:
> `:f>` should definitely be inside component, and we need to find a proper solution for `scroll-height` probably it needs to be implemented

`:f>` makes a component on its own. If we wrap it in another component - we will have 2 components) and no additional value.

I don't really see any problem in using `:>f` outside of the component, to be honest. Even if some developer will use `display-picture` without it, it will get a quite clear error explaining that he is trying to use hooks outside of the functional component. And since this is not even a part of the quo2 library there aren't many chances that it will be ever reused.

Reply:
We actually do need the `scroll-height` to be updated constantly, it is not enough to initialize it once. But we don't need the component to be re-created when `scroll-height` changed, it should be only re-rendered.
And as far as I can tell, this form creates full component recreation:
```
(defn display-picture
  [scroll-height cover]
  [:f>
   (fn []
     (let [input-range (if platform/ios? [-67 10] [0 150])
```

And the form you suggested won't react on `scroll-height` changes... and it is quite heavy for comprehension, I'd say.

Reply:
> and it is quite heavy for comprehension, I'd say.

it's a general approach in reagent, so if you're familiar with reagent it should be fine, more of that it's transparent, we clearly see that we don't want to use updated value in the render function

Reply:
honestly for me `[:f> display-picture @scroll-height cover-image]` looks like a hack, do we understand why `:f>` doesn't work inside `display-picture` ?  we need to understand that so we won't have this later in the code

Reply:
@ilmotta do you have any thoughts ?

Reply:
@flexsurfer, not really unfortunately. As I said, I don't have much to offer here... I'd need to spend time in the code to see if there's anything to be done. To me, as long as we treat `:f>` outside functions with hooks as the exception, I think we're fine. I take that this is an exception and that it leads to a better design from @vkjr's perspective.

Using `:f>` outside functions is relatively common tbh (not in our codebase though).

Reply:
yeah we don't have much experience with :f> so i just feel we might miss something important, performance issues etc , agree to find out we need to spend time and understand whats wrong with this code

Reply:
yep, and in this case it's both cleaner code and improves performance as it stops a lot of unnecessary rendering which was happen. (I did not benchmark this, but just assuming from how the screen was previously looking compared  with this pr)